src/http/middleware/_jwt.js

```diff
@@ -0,0 +1,26 @@
+
+// This lib is quite large at 339kb is there a better and lighter jwt decoder we could use
```

Comment:
339KB isn't too bad for a lib being used locally! However I've had really excellent results with [`jwt-simple`](https://www.npmjs.com/package/jwt-simple).